docs/user_guide/install_client.md

```diff
@@ -1,70 +1,128 @@
 # Installing Dragonfly Client
 
-You have three options when installing the Dragonfly client: installing from the latest package, installing from pulling the image or installing from the source code.
+This topic explains how to install the Dragonfly `dfclient`.
 
-## Installing from the Latest Package
+## Context
 
-You can install from the latest packages we provided.
+Install the `dfclient` in one of the following ways:
 
-1. Download a package of the client.
+- Deploying with Docker: **It is not recommended when you only need `dfget` for file distribution**.
```

Comment:
Could you please explain this more detailed in the doc?

Reply:
I will find a more suitable place to explain it.